README.md

```diff
@@ -296,4 +310,110 @@ The interesting approach with private and public variants is that you and your c
 Consumers are incited to use only public variants and it's recommended to communicate only on "official" and "public" variants  but if a custom specific need is required, consumers can use internal variants and customize the component as their needs. It's not recommended but sometimes, pragmatism is a good thing.
 
 
+### About variants versus props interpolation
+
+Variants is something relatively new in CSS-in-JS world that libraries like Stitches have democratized by adding  first-class variant API support.
+
+Stitches advocates [variants design instead of props interpolation)(https://stitches.dev/blog/migrating-from-emotion-to-stitches), meaning that variants are defined directly during the styles implementation, by infering definitions. It quicky adds complexity when it comes to extracting those variants in order to reuse them in another contexts. More generally, not having clear interfaces is rarely a good idea.
+
+Build-variants vision is more as:
+
+1. First, define clear interfaces for your components,
+2. Secondly, implement your interface by defining CSS and variants.
+3. Optionally, compose your variants if you need more high-level behaviors (like a "primary" type that defines a bunch of styles like colors, background and borders for example)
+
+Build-variants provides both, a first-class variant API and props interpolation, allowing to define variants according to props values.
+
+
+### About tokens
+
+[Tokens](https://stitches.dev/docs/tokens) (strings) could be seen as a handly way to create shortcuts for complex styles definitions. But you should consider as well the drawbacks of using simple strings that can't reference the source of the implementation in addition that adding more and more aliases of styles may obfuscate a bit which styles are really applied at the end.
+
+For values, you should better considerate importing directly what you need. If you need a custom set of styles, you can create a function and invokes it directly in styles definition.
```

Comment:
Another way to say it could be : 
"For values, you **may want to** consider importing..."

Typo - invok**e**